include/xen_blkif.h

```diff
@@ -130,4 +130,9 @@ static inline void blkif_get_x86_64_req(blkif_request_t *dst, blkif_x86_64_reque
 		dst->seg[i] = src->seg[i];
 }
 
+#define MAX_RING_PAGE_ORDER 3
```

Comment:
There is a reference to BLKIF_MAX_SEGMENTS_PER_REQUEST in drivers/td-ctx.c used in the blkif_get_req macro, does this need to change too?

Reply:
Damn its on the next page! Ignore me!